debug_toolbar/panels/sql/tracking.py

```diff
@@ -205,7 +214,7 @@ def _record(self, method, sql, params):
                 # case where Django can start a transaction before the first query
                 # executes, so in that case logger.current_transaction_id() will
                 # generate a new transaction ID since one does not already exist.
-                final_conn_status = conn.status
```

Comment:
Seems like the old logic was based on connection status, 
https://www.psycopg.org/docs/extensions.html#connection-status-constants

however according to the new api in psycopg3 `connection.info.status` we only have 2 states
https://www.psycopg.org/psycopg3/docs/api/pq.html#psycopg.pq.ConnStatus

Since `STATUS_IN_TRANSACTION` is never in any of the options of `connection.info.status` the test `tests.panels.test_sql.SQLPanelMultiDBTestCase.test_transaction_status` will always fail. How should we based the logic for the new states of `psycopg3`? should we rely on `conn.info.transaction_status` instead of `connection.info.status`